cts/cts-schemas.in

```diff
@@ -145,76 +141,64 @@ cleanup_module_error() {
     mv -- "$1.new" "$1"
 }
 
-# stdout: filename of the transformed file
-test_runner_upgrade() {
-    _tru_template=${1:?}
-    _tru_source=${2:?}  # filename
-    _tru_mode=${3:?}  # extra modes wrt. "referential" outcome, see below
+test_runner_upgrade_one() {
```

Comment:
I'm REALLY looking forward to having this script in Python, but I've been trying to avoid going down the rabbit hole of reimplementing it for now. Nonetheless, it might have been faster than trying to refactor a shell script...